### ISSUE TITLE
[BUG] Issue in document translation

### ISSUE THREAD

**Issue by kamahat** [bigtreetech/BIGTREETECH-Relay-V1.2#1]
Hello,

On the document "Relay V1.2 Operating Instruction.pdf", on chapter IV, the schematic is in chinese and not in english. This is not helpfull to understand what port/connector is used for.

**Comment by combolek** [comment]
I agree this needs to be fixed in the manual but in the meantime see this diagram:
![Wiring Diagram](https://user-images.githubusercontent.com/4743344/71552315-7fe5be80-29af-11ea-9c86-91ca32f1468f.png)

**Comment by kamahat** [comment]
> I agree this needs to be fixed in the manual but in the meantime see this diagram:
> ![Wiring Diagram](https://user-images.githubusercontent.com/4743344/71552315-7fe5be80-29af-11ea-9c86-91ca32f1468f.png)

Is this Wiring diagram all good ? 
the top left connector is labeled "Power failure detection diagnal line" as the last one but the 2nd and 3rd have the same labels "Signal/Ground". 
What are the good usage ?
1st connector : Power failur detection
2nd connector : Shutdown signal
3 rd : not sure ?

**Comment by combolek** [comment]
Yeah, the translation is poor. The top "power failure detection" really means "short circuit detection". I have not used it so I am not sure how it works but see issue #5 for example connections that other people did. There is other useful info in that thread too.

But the bottom "power failure detection" has literal meaning, i.e. detection when power goes out.

**Comment by carl1961** [comment]
I believe The top "power failure detection" it 5V in from the Mother Board, this may be used to feed the signal out on pin 3 , that a common procedure on many power resume modules. example pictures
![info611](https://user-images.githubusercontent.com/3056821/83820676-e83d7800-a692-11ea-952a-731e8a7eed90.png)

![info771](https://user-images.githubusercontent.com/3056821/83820666-e2e02d80-a692-11ea-9ad8-c2c60efa8f20.png)

**Comment by roetherb** [comment]
Just a note to BTT if they see this: Please avoid creating diagrams with black backgrounds. I spent entirely too much time trying to remove it before giving up and wasting black toner just to get a print for an assembly workstation.
